SConstruct

```diff
@@ -1338,6 +1338,23 @@ env.AlwaysBuild(
             env.PrettyCommand('TIDY', 'src', 'yellow')
         )))
 
+if 'PKG_CONFIG_PATH' in env.Dictionary():
+    pkg_config_path = env['PKG_CONFIG_PATH']
+elif 'PKG_CONFIG' in env.Dictionary() or env.Which('pkg-config'):
+    # https://linux.die.net/man/1/pkg-config the default is libdir/pkgconfig
+    pkg_config_path = os.path.join(env['ROC_SYSTEM_LIBDIR'], 'pkgconfig')
+else:
+    pkg_config_path = None
```

Comment:
Let's extract this pkg-config path detection to `cong.FindPkgConfigPath`. The method should be added to `site_scons/site_tools/roc/config.py` and used in SConstruct like this:

```
conf = Configure(env, custom_tests=env.CustomTests)

conf.FindPkgConfigPath()

env = conf.Finish()
```

Here is an example of such approach: https://github.com/roc-project/roc/blob/453631ccbd1cb9ac657877faef2997cee8a78d46/SConstruct#L478-L481